repo/config/discovery.go

```diff
@@ -2,6 +2,9 @@ package config
 
 type Discovery struct {
 	MDNS MDNS
+
+	//Routing sets default daemon routing mode.
+	Routing string
```

Comment:
I would make `Routing` its own top level section, and then give it a `Type` field (so we can add other parameters later)